text/exercise2/world_cup.md

```diff
@@ -23,14 +21,9 @@ the soccer field.
 
 Each edition of the Women's World Cup brings its own unforgettable moments 
 that etch themselves into the history of the sport. From stunning 
-touchdowns to nail-biting penalty shootouts, the tournament never fails to
+freekicks to nail-biting penalty shootouts, the tournament never fails to
 deliver drama and excitement. 
 
-As the tournament progresses, each match becomes a thrilling showcase of 
-talent, akin to the drama and intensity witnessed in NFL playoffs. With 
-each spectacular play and touchdown, the Women's World Cup cements its 
-status as a monumental event in the realm of global sports, leaving a 
-lasting legacy that echoes through the annals of both soccer and the NFL.
```

Comment:
Agree, best to remove this to avoid confusion between American Football / NFL and the WWC.